io500/sc23/aurora/aurora-reproducibility-productiony.md

```diff
@@ -0,0 +1,229 @@
+>   directories)
+> * File system config and tuning information (or a reason why this is not available due to lack
+>   of root access, etc) on each node type (e.g., non-default config parameters on all 3 Lustre
+>   MDS, OSS, and client)
+> * Any additional scripts utilized that impact IO500 execution beyond the io500 config file. For
+>   example, with IBM Spectrum Scale, the output of config, cluster, file system and fileset
+>   commands (and possibly even a dump of the configuration if possible). Each file system
+>   probably has similar type of config/tuning information that would need to be shared for a
+>   user to fully reproduce the environment.
+
+DAOS is configured to use 600 servers as described above and 300 Aurora compute nodes. DAOS clients will use all 8 NICs provided by each client for a total of 2400 client endpoints. The DAOS servers are configured to run 2 IO engines on each server for a total of 1200 daos servers. The underlying storage is configured by the `daos scm prepare` and `daos storage scan` as documented in the DAOS Adminstration Guide.
+
+The code is built with the Intel icc compiler, icc (ICC) 2021.9.0 20230201, and uses the Aurora MPICH MPI pre-release.
+The Aurora MPICH is built from the open source [https://github.com/pmodels/mpich|MPICH].
+
+The code used in this run is from the commit hash, 9597af34c30be0ea4049e43c16169057227e0527.
+
+The io500 script is executed with these environment variables.
+```
+export CRT_MRC_ENABLE=1
+export FI_CXI_RX_MATCH_MODE=hybrid
+export FI_CXI_REQ_BUF_SIZE=8388608
+export FI_CXI_OFLOW_BUF_SIZE=8388608
+export FI_CXI_REQ_BUF_MIN_POSTED=8
+export FI_CXI_DEFAULT_CQ_SIZE=131072
+export FI_CXI_CQ_FILL_PERCENT=20
+export MPIR_CVAR_BCAST_POSIX_INTRA_ALGORITHM=mpir
+export MPIR_CVAR_ALLREDUCE_POSIX_INTRA_ALGORITHM=mpir
+export MPIR_CVAR_BARRIER_POSIX_INTRA_ALGORITHM=mpir
+export MPIR_CVAR_REDUCE_POSIX_INTRA_ALGORITHM=mpir
+export LD_LIBRARY_PATH=/scratchbox/daos/mschaara/install/mpich-52.2/lib/:$LD_LIBRARY_PATH
+export PATH=/scratchbox/daos/mschaara/install/mpich-52.2/bin/:$PATH
+```
+
+The server YAML file is provided at [].
+
+## FAULT TOLERANCE MECHANISMS
+
+> Does your system have a single point of failure? Please describe all mechanisms provide fault
+> tolerance for the submitted storage system. Be specific to your submission, not general storage
+> system capabilities.
+> 
+> * Power
+> * Networking (e.g., dual redundant switches)
+> * Inter Storage data and metadata server (e.g., active-active servers, client-directed RAID,
+>   declustered RAID, erasure-coding, 3-way replication)
+> * Intra Storage data and metadata server storage media (e.g., raid5)
+> 
+> Please list any additional information needed to determine whether this system has a single point
+> of failure.
+
+The Aurora hardware and software storage system is able to provide full fault tolerance at various levels of reduendancy depedending on the component.
+* Power
```

Comment:
add blank line before this, so md renders the bullet list correctly

Reply:
corrected.